schemas/yaml/Utterance.yml

```
@@ -84,6 +84,17 @@ properties:
     format: uri
     description: 'A URL where a presentational format for this resource may be viewed'
 
+  literalTranslation:
+    title:  Literal Translation
+    type:   object
+    $ref:   'http://schemas.digitallinguistics.io/Translation.json'
+    description: 'The literal translations for this Utterance, optionally in multiple languages. The translation must be provided in at least one language.'
+
+  phonetic:
+    title:  Phonetic 
+    type:   string
+    description: 'The phonetic transcription for this Utterance in IPA. The transcription should not include phonetic brackets.'
```

Although we can't enforce this programmatically, we can still require that this field contain only valid IPA characters. (Btw, I was thinking about this more, and I think it would be essentially impossible to do this programmatically. The combinations of diacritics are endless.)

```suggestion
    description: 'The phonetic transcription for this Utterance in IPA. Only valid IPA characters are allowed. The transcription should not include phonetic brackets.'
```

---

Okay. I assume that will be a separate project then. I agree that some sort of IPA repo would be convenient, but challenging.